tests/validator/view-schema/v2-test.js

```diff
@@ -32,79 +31,177 @@ describe('v2 schema validation', () => {
-        value.cells[0].renderer.options.none.value = 1
-        expect(schemaValidator.validate(value, model)).to.equal(true)
-      })
-      it('passes validation when value is boolean', function () {
-        value.cells[0].renderer.options.none.value = true
-        expect(schemaValidator.validate(value, model)).to.equal(true)
-      })
-      it('passes validation when value is object', function () {
-        value.cells[0].renderer.options.none.value = {}
-        expect(schemaValidator.validate(value, model)).to.equal(true)
-      })
+        it('passes validation when value is number', function () {
+          value.cells[0].renderer.options.data[0].value = 1.1
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
 
-      it('fails validation when value is an array', function () {
-        value.cells[0].renderer.options.none.value = []
-        expect(schemaValidator.validate(value, model)).to.equal(false)
+        it('passes validation when value is integer', function () {
+          value.cells[0].renderer.options.data[0].value = 1
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+        it('passes validation when value is boolean', function () {
+          value.cells[0].renderer.options.data[0].value = true
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+        it('passes validation when value is object', function () {
+          value.cells[0].renderer.options.data[0].value = {}
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+
+        it('fails validation when value is an array', function () {
+          value.cells[0].renderer.options.data[0].value = []
+          expect(schemaValidator.validate(value, model)).to.equal(false)
+        })
+
+        it('fails validation when value is null', function () {
+          value.cells[0].renderer.options.data[0].value = null
+          expect(schemaValidator.validate(value, model)).to.equal(false)
+        })
       })
 
-      it('fails validation when value is null', function () {
-        value.cells[0].renderer.options.none.value = null
-        expect(schemaValidator.validate(value, model)).to.equal(false)
+      describe('when none value is specified', function () {
+        it('passes validation when value is string', function () {
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+
+        it('passes validation when value is number', function () {
+          value.cells[0].renderer.options.none.value = 1.1
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+
+        it('passes validation when value is integer', function () {
+          value.cells[0].renderer.options.none.value = 1
+          expect(schemaValidator.validate(value, model)).to.equal(true)
+        })
+        it('passes validation when value is boolean', function () {
```

Comment:
Agreed, I copy/pasted code written by @sophypal and that isn't something we lint so I've not been to pressing on it. I'm also not sure if we've ever documented that as part of our styling standards or not.